src/hypothesis/internal/conjecture/engine.py

```diff
@@ -107,30 +113,11 @@ def __init__(
 
         self.shrunk_examples = set()
 
-        self.discovery_queue = deque()
+        self.tag_intern_table = {}
```

Comment:
Nothing specific in mind, but how does this relate to the recent (and ongoing) caching work?